Python/pylifecycle.c

```diff
@@ -412,10 +412,11 @@ _Py_InitializeEx_Private(int install_sigs, int install_importlib)
     if (interp->sysdict == NULL)
         Py_FatalError("Py_Initialize: can't initialize sys dict");
     Py_INCREF(interp->sysdict);
+
+    PySys_SetPath(Py_GetPath());
```

Comment:
Consider adding a comment here about how Py_GetPath() must be called before the flags get set.